WordPress/src/main/java/org/wordpress/android/ui/posts/PostListAction.kt

```diff
@@ -32,7 +90,27 @@ fun handlePostListAction(activity: FragmentActivity, action: PostListAction) {
             ActivityLauncher.addNewPostForResult(activity, action.site, action.isPromo)
         }
         is PostListAction.PreviewPost -> {
-            ActivityLauncher.viewPostPreviewForResult(activity, action.site, action.post)
+            if (action.post.isPage) {
+                ActivityLauncher.viewPostPreviewForResult(activity, action.site, action.post)
+            } else {
+                val opResult = remotePreviewLogicHelper.runPostPreviewLogic(
+                        activity = activity,
+                        site = action.site,
+                        post = action.post,
+                        helperFunctions = getPostsListStrategyFunctions(activity, action)
+                )
+
+                if (opResult == RemotePreviewLogicHelper.PreviewLogicOperationResult.OPENING_PREVIEW) {
+                    // TODO: evaluate if this case can really happen in the Posts List.
```

Comment:
AFAIU (a confirmation from your side is more tha welcome of course 😊) currently the PREVIEW button is available in a post item in the Posts List only if isLocalDraft or isLocallyChanged (https://github.com/develric/WordPress-Android/blob/010931d67d965d2406f8bbf40b1829f91107d7b5/WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiStateHelper.kt#L286). So in this PR, Draft, Published and Scheduled posts will not get here. Trashed posts can reach here currently, but I think they will be managed differently soon based on this #10106.
So i would keep for now the if check here but changing the TODO comment for example as in here a05aba815ad943d5430d38074ef54ae1baace40c. wdyt?
Pinging also @maxme on this since he triggered the above ticket.

Reply:
The updated comment makes it more clear and actionable even for someone who doesn't know the code so well -> 👍. I'd also consider updating the description of #10106 with a note to remove this if statement, so we don't forget about it. Wdyt?

Reply:
I agree we should keep the TODO here - note that we'll probably merge this PR to its own feature branch once we solve some known issues. Good point, @malinajirka we should comment on #10106 to make sure we don't forget it.

Reply:
Added note about this in #10106 description 👍.